docs/daily-usage.md

```diff
@@ -103,6 +106,9 @@ invoke --list
 To browse Odoo go to `http://localhost:${ODOO_MAJOR}069` (i.e. for Odoo 11.0 this would
 be `http://localhost:11069`).
 
+**Note:** For more information, check
+[Visual Studio Code Integration](#visual-studio-code-integration)
```

Comment:
This note should go above, in this paragraph: https://github.com/Tecnativa/doodba-copier-template/pull/115/files#diff-db5b9c6fa770aa40cf2f944e5129d29dR71-R72